### DISCUSSION TITLE
Sync all shared business folders with single line command

### DISCUSSION THREAD

**Discussion by em-oz** [abraunegg/onedrive#2140] [General]
Hi, I was wondering if there is a way to sync all shared folders with a single option. Otherwise I need to update 'business_shared_folders' config file every time a new shared folder is added to the list, I guess.

**Comment by abraunegg** [comment]
@em-oz 
Correct - you need to update 'business_shared_folders' file with each new folder you want to sync.

There is no option to blindly sync all.
